src/electron/electron/git.cljs

```diff
@@ -51,7 +51,7 @@
           p (.join path graph-path ".git")]
       (when (and (fs/existsSync p)
                  (.isFile (fs/statSync p)))
-        (let [content (.toString (fs/readFileSync p))
+        (let [content (string/trim (.toString (fs/readFileSync p)))
```

Comment:
From @cnrpman:
> the git file has a newline at the end. When reading the path, it will read an extra carriage return. Because there was no trim, the original remove function thinks that the path is inconsistent with the new separate git dir, and the .git file will be deleted.